src/main/java/xbot/common/controls/sensors/mock_adapters/MockSettableTimer.java

```diff
@@ -0,0 +1,44 @@
+package xbot.common.controls.sensors.mock_adapters;
+
+import com.google.inject.Singleton;
+
+import xbot.common.controls.sensors.XSettableTimerImpl;
+
+/**
+ * A settable timer for use in simulations
+ */
+@Singleton
+public class MockSettableTimer implements XSettableTimerImpl {
```

Comment:
This is basically exactly the same as MockTimer but with an interface to make the setters accessible. Should I just update MockTimer itself?

Reply:
That makes sense - I can't think of a reason we'd want to use the older MockTimer given that the settable one just makes it easier to use when running tests.